cdap-api-common/src/main/java/io/cdap/cdap/api/data/schema/Schema.java

```diff
@@ -112,7 +112,8 @@ public boolean isSimpleType() {
    * negative scale is an int representing number of zeros appended to decimal number.
    *
    * For example,
-   * 10.2 number will have precision as 3 as scale as 1.
+   * 102 number with precision 3 and scale 1 will be represented as 10.2.
```

Comment:
An unscaled value of 102 and scale 1 represents the number 10.2 with precision 3.
An unscaled value of 111 and scale -1 represents the number 1110 with precision 3.

Reply:
though for the second example, isn't the precision 4?

Reply:
If unscaled value is `111`, and scale is negative zeros will be appended.
![image](https://user-images.githubusercontent.com/14131070/57398690-7c819680-7184-11e9-9fbf-3b0ae36ca6b5.png)

```
precision: 3
scale: -1
unscaled value: 111
Plain string: 1110